api2/pkg/auth/handler.go

```diff
@@ -33,7 +40,12 @@ func init() {
 	gob.Register(&Profile{})
 }
 
-func NewHandler(ctx context.Context, issuerURL, clientID, clientSecret, redirectURL string, store sessionmanager.Store) (*Handler, error) {
+func NewHandler(
```

Comment:
We should introduce a consistent rule for how to format params to functions like this. Is it possible to make that a team-wide editor setting as well @ludydoo ?

Reply:
I think Go comes with it's own formatting tool `go fmt` that should handle most cases like these. In goland you can format a file with it using `Ctrl+Alt+Shift+F`.